docs/running-on-yarn.md

```diff
@@ -68,9 +68,9 @@ In YARN terminology, executors and application masters run inside "containers".
 
     yarn logs -applicationId <app ID>
     
-will print out the contents of all log files from all containers from the given application. You can also view the container log files directly in HDFS using the HDFS shell or API. The directory where they are located can be found by looking at your YARN configs (`yarn.nodemanager.remote-app-log-dir` and `yarn.nodemanager.remote-app-log-dir-suffix`).
+will print out the contents of all log files from all containers from the given application. You can also view the container log files directly in HDFS using the HDFS shell or API. The directory where they are located can be found by looking at your YARN configs (`yarn.nodemanager.remote-app-log-dir` and `yarn.nodemanager.remote-app-log-dir-suffix`). The logs are also available on Web UI. You need have both the Spark history server and the MR history server running and configure `yarn.log.server.url` in yarn-site.xml properly. The log url on the Spark history server UI will redirect you to the MR history server to show the aggregated logs.
```

Comment:
Minor text updates: MR history server -> MapReduce History Server. I'd hyperlink one of them too. url -> URL. back-tick `yarn-site.xml` It would be nice to link through to YARN docs that would describe these parameters too, if possible.